dio/dartdoc_options.yaml

```diff
@@ -0,0 +1,3 @@
+dartdoc:
```

Comment:
What if we do a symlink to the dart folder for all packages? they are all the same.